website/docs/components/code-editor/partials/guidelines/guidelines.md

```diff
@@ -104,3 +104,22 @@ In **Figma** we provide a handful of example languages intended as visual exampl
 
 If you wish to create custom examples using the Code Editor, we publish all of the relevant syntax highlighting variables in the [HDS Components v2.0](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?node-id=67166-37020&t=gWdKy44MzTP4cTRo-1) library. However, due to the number of languages supported by the component, the color variables use a generic naming schema (e.g., cyan, red, purple) to remain as agnostic as possible when being applied to different languages.
 For more details around syntax, visit the [specifications](/components/code-editor?tab=specifications).
+
+## Linting
+The Code Editor supports linting for JSON using [CodeMirror6](https://codemirror.net/examples/lint/). This feature highlights an error with an underline and displays an icon next to the line number, each of which shows a tooltip on hover. To view all alerts in the editor, users can open a dialog using Ctrl-Shift-m (Cmd-Shift-m on macOS).
+
+!!! Info 
+This component is only available in Ember and not currently supported in Figma.
+!!!
+
+![Code Editor linting with errors present and a tooltip displayed](/assets/components/code-editor/codeeditor-linting-preview-tooltip.png)
+
+When linting is enabled, the CodeEditor will have a minimum height set by default to avoid the alert dialog covering all the content within the editor when opened. The minumum height for the CodeEditor with linting is 160px. The linting alert dialog is always 80px in height.
+
+![Code Editor with linting enabled and errors present](/assets/components/code-editor/codeeditor-linting-preview.png)
```

Comment:
Same as the previous image, the paragraph above does a great job explaining the same information as the image.

```suggestion
![](/assets/components/code-editor/codeeditor-linting-preview.png)
```